verticapy/learn/vmodel.py

```diff
@@ -4663,18 +3950,17 @@ def fit(self, input_relation: Union[str, vDataFrame], X: list = []):
         elif self.type == "Normalizer":
             query += ", {}".format(parameters["method"])
             del parameters["method"]
-        if self.type not in ("Normalizer", "MCA",):
+        if self.type not in ("Normalizer", "MCA"):
             query += " USING PARAMETERS "
         if (
             "init_method" in parameters
             and not (isinstance(parameters["init_method"], str))
             and self.type in ("KMeans", "BisectingKMeans")
         ):
-            schema = schema_relation(self.name)[0]
-            name = "VERTICAPY_KMEANS_INITIAL_{}".format(get_session(self.cursor))
+            name_init = gen_tmp_name(schema=schema_relation(self.name)[0], name="kmeans_init")
             del parameters["init_method"]
             try:
-                self.cursor.execute("DROP TABLE IF EXISTS {}.{}".format(schema, name))
+                drop_if_exists(name_init, method="table")
             except:
                 pass
```

Comment:
Can we get ride of this `pass`?

Reply:
Done; The change will be made in the next commit.